src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -25,12 +25,7 @@ void func_80A99384(EnKo* this, GlobalContext* globalCtx);
 void func_80A99438(EnKo* this, GlobalContext* globalCtx);
 void func_80A99504(EnKo* this, GlobalContext* globalCtx);
 void func_80A99560(EnKo* this, GlobalContext* globalCtx);
-Gfx* EnKo_SetEnvColor(GraphicsContext* gfxCtx, u8 r, u8 g, u8 b, u8 a);
-s32 EnKo_OverrideLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx,
-                          Gfx** arg6);
-u16 func_80A97610(GlobalContext*, Actor*);
-s16 func_80A97738(GlobalContext*, Actor*);
-f32 func_80A97BC0(EnKo* this);
+
```

Comment:
```suggestion
```
empty line, fig cmon you had one job :feelsbadman:

Reply:
this was on purpose, the action funcs in a group makes sense to me